ziti-fabric/subcmd/snapshot_db.go

```diff
@@ -42,13 +42,13 @@ func snapshotDb(cmd *cobra.Command, args []string) {
 		panic(err)
 	}
 
-	requestMsg := channel2.NewMessage(int32(mgmt_pb.ContentType_SnapshotDbRequestType), nil)
-	responseMsg, err := ch.SendAndWaitWithTimeout(requestMsg, 5*time.Second)
+	requestMsg := channel.NewMessage(int32(mgmt_pb.ContentType_SnapshotDbRequestType), nil)
+	responseMsg, err := requestMsg.WithTimeout(5 * time.Second).SendForReply(ch)
```

Comment:
5 seconds might not be long enough to snapshot a big db? maybe this one should be bigger?

Reply:
It'll only timeout the response, it won't halt the operation. Ideally it should be configurable from the command line. However, ziti-fabric is pretty close to deprecated, so I didn't think it was worth the effort